src/main/java/com/checkmarx/flow/custom/GitHubIssueTracker.java

```diff
@@ -63,29 +63,41 @@ public void init(ScanRequest request, ScanResults results) throws MachinaExcepti
      */
     @Override
     public List<Issue> getIssues(ScanRequest request) {
-        log.info("Executing getIssues GitHub API call");
+        String apiUrl = String.format("%s/%s/%s/issues?state=all&per_page=%s",
+                properties.getApiUrl(),
+                request.getNamespace(),
+                request.getRepoName(),
+                ISSUES_PER_PAGE);
+
+        log.info("Executing getIssues GitHub API call: {}", apiUrl);
         List<Issue> issues = new ArrayList<>();
-        HttpEntity httpEntity = new HttpEntity<>(createAuthHeaders());
-        String apiUrl = properties.getApiUrl().concat("/").concat(request.getNamespace().concat("/")).concat(request.getRepoName()).concat("/issues?state=all&per_page=").concat(ISSUES_PER_PAGE);
+        HttpEntity<?> httpEntity = new HttpEntity<>(createAuthHeaders());
+
         ResponseEntity<com.checkmarx.flow.dto.github.Issue[]> response = restTemplate.exchange(apiUrl,
                 HttpMethod.GET, httpEntity, com.checkmarx.flow.dto.github.Issue[].class);
-        if(response.getBody() == null){
+
+        if (response.getBody() == null) {
+            log.debug("No issues found.");
```

Comment:
Won't it also be valuable to know that no issues were found in INFO level?

Reply:
Updated